app/views/events/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h2>Event created by <%= @event.creator.name %> </h2>
+<%= @event.title %><br>
+<br><%= @event.date %>
+
+<ul>
+  <% @event_attendees.errors.full_messages.each do |msg| %>
```

Comment:
- **_This is throwing an error whenever I clicked the "Check Event" button. Check the image :point_down:_**

![Screenshot from 2020-08-31 10-34-11](https://user-images.githubusercontent.com/48312803/91744205-b72fef00-eb7e-11ea-8b03-fec4b3a8007b.png)

Reply:
Thanks for reviewing this fast Temitayo! I'll fix that right now.